### PR TITLE
Jk/feature/apollo setup

### PR DESCRIPTION
**Pull request by JeffKersting**
#### Purpose:
 > Check all that apply.
- [x] Feature
- [ ] Refactor
- [ ] Bug Fix 
- [ ] Clean Up

### What does this PR do?
 > Clearly and concisely describe changes made. 

Sets up Apollo client and provider. Adds query to user landing page.

Issues closed #

Initial user landing page query resulted in error when used in use effect. 
Solution: add conditional check to use effect to check success of query.

> Describe your code changes in detail for reviewers.

Adds Apollo client to index.js, creates a new GraphQL folder, with a query and mutation file. Adds get user query to query file.

Index.js. GraphQL/queries. User-Landing.js.

Next steps: setup mutations.

### CODE REVIEW

File: src/GraphQL/queries.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import {gql} from '@apollo/client'
+
+export default gql`
+  query {
+    user(id: 2) {
```
Comment (gaj23) [inline src/GraphQL/queries.js:5]: I think we should talk to Robbie about this! It feels like a blocker.

File: src/components/App/App.js [hunk 2]
```diff
@@ -34,6 +35,8 @@ const App = () => {
       <Route exact path="/create-postcard" render={() => <PostcardForm />} />
```
Comment (gaj23) [inline src/components/App/App.js:35]: These should be reformatted for readability.

File: src/components/User-Landing/User-Landing.js [hunk 3]
```diff
@@ -1,135 +1,99 @@
...
+
+  if (loading) return null
+  if (error) return (<Redirect to='/Error' />)
 
   return (
-    <section className="landing-wrapper">
-      {isLoggedIn && (
-        <article>
+    <div className='landing-wrapper'>
+      {!updating &&
```
Comment (gaj23) [inline src/components/User-Landing/User-Landing.js:43]: I think we should includes isLoggedIn & !updating

**Review by gaj23 — APPROVED** [review]
Great. 
There's some strange formatting things that I think are happening from us all working on different branches & at different paces.

**Review by gaj23 — APPROVED** [review]

File: src/components/User-Landing/User-Landing.js [hunk 4]
```diff
@@ -1,135 +1,99 @@
...
 import UpdateUserInfoForm from '../Update-Form/Update-Form';
+import { useQuery } from '@apollo/client'
+import GET_USER from '../../GraphQL/queries'
+
+const UserLandingPage = () => {
+
+  const { error, loading, data } = useQuery(GET_USER, {variables: {id: 2}})
+
+
+  const [user, setUser] = useState({})
```
Comment (kelsiebesingeryeh) [inline src/components/User-Landing/User-Landing.js:12]: I usually set empty objects as null, not sure if it matters but wanted to point it out

**Review by kelsiebesingeryeh — APPROVED** [review]

**Pull request merged**